lib/samples/set_up_location_driven_geotriggers/set_up_location_driven_geotriggers.dart

```diff
@@ -0,0 +1,611 @@
+                          SizedBox(
+                            height: 250,
+                            width: MediaQuery.of(context).size.width * 0.75,
+                            child: WebViewWidget(
+                              controller: featureWebViewMap[feature]!,
+                            ),
+                          ),
+                        ],
+                      );
+                    },
+                  );
+                },
+              ),
+            ),
+            TextButton(
+              onPressed: () {
+                Navigator.pop(context);
+              },
+              child: const Text('Close'),
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+
+  Future<void> onMapViewReady() async {
+    // Create the map based on a webmap and add it to the MapView
+    final map = ArcGISMap.withUri(
+      Uri.parse(
+        'https://www.arcgis.com/home/item.html?id=6ab0e91dc39e478cae4f408e1a36a308',
+      ),
+    );
+    _mapViewController.arcGISMap = map;
+
+    // Set the location data source to the MapView's LocationDisplay
+    _mapViewController.locationDisplay.dataSource = _locationDataSource;
+    await _locationDataSource.start();
+
+    // Setup the geotriggers
+    await _setupGeotriggers();
+
+    // Set the ready state variable to true to enable the sample UI.
+    setState(() => _ready = true);
+  }
+
+  // Sets up the Geotriggers and listens for Geotrigger events.
+  Future<void> _setupGeotriggers() async {
+    // Setup the points of interest Geotrigger
+    await createGeotriggerMonitor(
+      featureTable: _gardenPoisTable,
+      bufferSize: 10,
+      name: _poiGeotriggerName,
+    );
+
+    // Set up the sections Geotrigger
+    await createGeotriggerMonitor(
+      featureTable: _gardenSectionsTable,
+      name: _sectionGeotriggerName,
+    );
+  }
+
+  // Creates a GeotriggerMonitor using the provided ServiceFeatureTable and
+  // optional buffer size. The stream for changing events will be listened to,
+  //and the monitor started.
+  Future<void> createGeotriggerMonitor({
+    required ServiceFeatureTable featureTable,
+    required String name,
+    double bufferSize = 0.0,
+  }) async {
+    // Set up the fence parameters
+    final fenceParameters = FeatureFenceParameters(
+      featureTable: featureTable,
+      bufferDistance: bufferSize,
+    );
+
+    // Create the geotrigger. The Arcade expression provides a convinient way to
+    // get the name of the triggering feature.
+    final geotrigger = FenceGeotrigger(
+      feed: LocationGeotriggerFeed(locationDataSource: _locationDataSource),
+      ruleType: FenceRuleType.enterOrExit,
+      fenceParameters: fenceParameters,
+      messageExpression: ArcadeExpression(expression: r'$fencefeature.name'),
+      name: name,
+    );
+
+    // Create the GeotriggerMonitor and listen to the onGeotriggerNotificationEvent stream
+    final monitor = GeotriggerMonitor.withGeotrigger(geotrigger);
```

Comment:
```suggestion
    final monitor = GeotriggerMonitor(geotrigger);
```
I think this will need to be updated (we made it an unnamed constructor)

Reply:
Fixed. My arcgis_maps package was a bit out of date.